addons/website_sale/models/product_template.py

```diff
@@ -242,16 +244,16 @@ def _get_combination_info(self, combination=False, product_id=False, add_qty=1,
             quantity_1 = 1
             combination_info['price'] = self.env['account.tax']._fix_tax_included_price_company(
                 combination_info['price'], product_taxes, taxes, company_id)
-            price = taxes.compute_all(combination_info['price'], pricelist.currency_id, quantity_1, product, partner)[tax_display]
+            price = taxes.compute_all(combination_info['price'], current_website.currency_id, quantity_1, product, partner)[tax_display]
```

Comment:
Not 100% sure you can rely on pricelist == website.pricelist_id (if pricelist is given).
Might be more safe to say `currency = pricelist.currency_id or website.currency_id` :thinking: